src/Utils/FileUtils.cpp

```diff
@@ -238,15 +238,17 @@ fs::path FileUtils::getPreferredPath(const fs::path& path) {
 
 std::string FileUtils::hashPath(const fs::path& path) {
   const std::string separator(1, fs::path::preferred_separator);
-  std::string hashedpath;
-  std::size_t val = std::hash<std::string>{}(path.string());
   std::string last_dir = path.string();
-  if (!last_dir.empty()) last_dir.erase(last_dir.end() - 1);
-  auto it1 = std::find_if(last_dir.rbegin(), last_dir.rend(),
-                          [](char ch) { return (ch == '/' || ch == '\\'); });
-  if (it1 != last_dir.rend()) last_dir.erase(last_dir.begin(), it1.base());
-
-  hashedpath = last_dir + "_" + std::to_string(val) + separator;
+  std::size_t val = std::hash<std::string>{}(last_dir);
```

Comment:
Would probably a good opportunity to add a unit test for hashPath() in `FileUtils_test.cpp`

Reply:
Hmm I have got my hands (and my head) full at the moment. Don't wish to sign up for anything new until I make some room on my plate. I am still investigating the regression script stalling and I am hitting walls on every direction I turn. May be next time. Sorry!